src/templates/views/my-past-trips.njk

```diff
@@ -0,0 +1,44 @@
+{% include "common/site-head.njk" %}
+<title>My Past Trips - DOC Trailhead</title>
+<link rel="stylesheet" href="/static/css/all-trips.css?id=2">
+<link rel="stylesheet" href="/static/css/common.css?id=2">
+{% include "common/site-nav.njk" %}
+<main>
+<section class="info-card">
+<nav class=my-trips-nav view-types>
+    <ul>
+        <li><a href="/my-trips"><img src="/static/icons/mountain-icon.svg">Upcoming Trips</a></li>
+        <li><a href="/my-past-trips"><img src="/static/icons/trip-icon.svg">Past Trips</a></li>
+    </ul>
+    <div class=filters>
+        <label>
+            <input
+                id="showMyPastTrips"
+                name="showMyPastTrips"
+                hx-get="?view={{ view }}&leaderOnly={{ 'false' if leader_only else 'true' }}&showAll={{ 'false' if show_all else 'true' }}"
+                hx-push-url=true
+                hx-target="body"
+                type="checkbox"
+                autocomplete="off"
+                {{ 'checked' if leader_only }} 
+                onchange="this.setAttribute('hx-get', this.checked ? '?view={{ view }}&leaderOnly=true&showAll=false' : '?view={{ view }}&leaderOnly=false&showAll=true')"
+            >
+            Led by Me
+        </label>
+    </div>
+</nav>
+</section>
+
+<section class="trips">
+    {% for trip in trips %}
+    <a href="/{% if trip.leader === 1 %}leader/{% endif %}trip/{{ trip.id }}" class=trip-card>
```

Comment:
I saw this and said "there's no way that's how I handled this" and then looked at `my-trips` and, yup, that's exactly how I handled it.